node-tests/unit/utilities/configuration-reader-test.js

```diff
@@ -90,6 +90,23 @@ describe('ConfigurationReader', function() {
         });
       }).to.throw('Cannot load configuration file \'' + path.join(root, configPath) + '\'. Note that the default location of the ember-cli-deploy config file is now \'config/deploy.js\'');
     });
+
+    it('raises an error if a config doesn\'t exist for the current environment', function() {
+      var configPath = './node-tests/fixtures/config/deploy.js';
+
+      try {
+        new ConfigurationReader({
+          configFile: configPath,
+          ui: ui,
+          project: project,
+          environment: 'non-existent-env'
+        });
+
+        expect(false).to.be.ok;
```

Comment:
Have never really known what the best practice is here when asserting that an Error is thrown.  Any suggestions would be much appreciated.